src/storage/table_codec.rs

```diff
@@ -98,6 +98,45 @@ impl TableCodec {
                     })
             })
     }
+
+
+    /// Key: RootCatalog_0_TableName
+    /// Value: TableName
+    pub fn encode_root_table(table_name: &str) -> Option<(Bytes, Bytes)> {
+        let key = format!(
+            "RootCatalog_{}_{}",
+            BOUND_MIN_TAG,
+            table_name,
+        );
+
+        bincode::serialize(&table_name).ok()
+            .map(|bytes| {
+                (Bytes::from(key.into_bytes()), Bytes::from(bytes))
+            })
+    }
+
+    pub fn decode_root_table(key: &[u8], bytes: &[u8]) -> Option<(String,String)> {
+        String::from_utf8(key.to_owned()).ok()?
+            .split("_")
+            .nth(2)
+            .and_then(|table_name| {
```

Comment:
value may be used to store meta information, but the table name should not be repeated.
At the same time, it is best to note that value is reserved for saving meta-information.